components/globals/Base.js

```diff
@@ -38,7 +39,15 @@ const Base = ({ children }) => {
             {isAdmin && <AdminNav user={children.props.user} />}
           </header>
         )}
-        <main id="content">{children}</main>
+
+        <main id="content">
+          {formConfig?.securityAttribute && (
+            <SecurityAttributeBadge
```

Comment:
I wonder if this is the right place to add the security attribute badge. Should not this be part of the Form component directly? The `Base` component is also used in non form related pages so it should probably stay clean and only includes basic things. What do you think?

Reply:
One reason i put it there was that i thought it make much more sens to have the badge above the form's title instead of . Do you see 
![Screen Shot 2022-05-03 at 2 51 25 PM](https://user-images.githubusercontent.com/3014890/166522659-67be1ae4-285a-461e-9e6a-cfe454f3dea1.png)

Reply:
Gotcha! Then it can still be done through the `DynamicForm.tsx` (which is closer to the form component than `Base`) if I am correct ;)

Reply:
That's right 👍🏿  I've forgotten that a form is wrapped with DynamicFrom. Ty